x-pack/plugins/security_solution/public/app/home/global_header/index.tsx

```diff
@@ -37,63 +37,75 @@ const BUTTON_ADD_DATA = i18n.translate('xpack.securitySolution.globalHeader.butt
  * This component uses the reverse portal to add the Add Data, ML job settings, and AI Assistant buttons on the
  * right hand side of the Kibana global header
  */
-export const GlobalHeader = React.memo(
-  ({ setHeaderActionMenu }: { setHeaderActionMenu: AppMountParameters['setHeaderActionMenu'] }) => {
-    const portalNode = useMemo(() => createHtmlPortalNode(), []);
-    const { theme } = useKibana().services;
-    const { pathname } = useLocation();
+export const GlobalHeader = React.memo(() => {
+  const portalNode = useMemo(() => createHtmlPortalNode(), []);
+  const { theme, setHeaderActionMenu, i18n: kibanaServiceI18n } = useKibana().services;
+  const { pathname } = useLocation();
 
-    const getTimeline = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
-    const showTimeline = useShallowEqualSelector(
-      (state) => (getTimeline(state, TimelineId.active) ?? timelineDefaults).show
-    );
+  const getTimeline = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
+  const showTimeline = useShallowEqualSelector(
+    (state) => (getTimeline(state, TimelineId.active) ?? timelineDefaults).show
+  );
 
-    const sourcererScope = getScopeFromPath(pathname);
-    const showSourcerer = showSourcererByPath(pathname);
+  const sourcererScope = getScopeFromPath(pathname);
+  const showSourcerer = showSourcererByPath(pathname);
+  const dashboardViewPath =
+    matchPath(pathname, {
+      path: `/${SecurityPageName.dashboards}/:id`,
+      exact: false,
+      strict: false,
+    }) != null;
```

Comment:
Can we move this logic to an `isDashboardViewPath` helper? similar to the `isDetectionsPath` called below. 
Also, better use `${DASHBOARDS_PATH}/:id`, the `SecurityPageName` is not the path.